store/driver/tikv_driver.go

```diff
@@ -24,6 +24,7 @@ import (
 	"time"
 
 	"github.com/pingcap/errors"
+	tidbconfig "github.com/pingcap/tidb/config"
```

Comment:
I'm afraid that you may need to check if it breaks the work about extracting the tikvclient out from tidb repo.
cc @disksing